code401Challenges/src/test/java/code401Challenges/linkedlist/LinkedListTest.java

```diff
@@ -0,0 +1,59 @@
+package code401Challenges.linkedlist;
+
+import org.junit.Test;
+import sun.awt.image.ImageWatched;
```

Comment:
You don't need this import statement! In fact, I can't even run your tests with this here!